### DISCUSSION TITLE
Add U8g2 frame into EdgeTX

### DISCUSSION THREAD

**Discussion by NSG-666** [EdgeTX/edgetx#4051] [Ideas]
Add Chinese language support for OLED RC (use U8g2 frame), and build a 
customized startup page.
This is a video:[](https://youtu.be/jKuEs4jtMwg)
https://youtu.be/jKuEs4jtMwg

**Comment by raphaelcoeffic** [comment]
> Add Chinese language support for OLED RC (use U8g2 frame), and build a customised startup page.

Nice! Where is the code?

**Reply by NSG-666** [reply]
@raphaelcoeffic in order to make hardware debugging more convenient, I move the dev env to cube IDE (i think i will try a way to make a PR later), now it is looks like that:
1、
![image](https://github.com/EdgeTX/edgetx/assets/18585840/ecc96900-ee7e-42b4-8c7e-73ed212af25c)
2、
![debug](https://github.com/EdgeTX/edgetx/assets/18585840/660afe7e-8c79-4240-a8c9-43d34a193453)
3、
![debug1](https://github.com/EdgeTX/edgetx/assets/18585840/52598b04-ecf3-4598-a139-5bcedbc165b1)

**Reply by raphaelcoeffic** [reply]
Thx! Yes, please make a PR so we can assess the code changes. It sound quite fun indeed.
